src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -904,12 +903,15 @@ export default class Aircraft {
 
         let isExpeditingString = '';
         if (expedite) {
-            isExpeditingString = 'and expedite';
+            isExpeditingString = ' and expedite';
```

Comment:
I don't like these leading spaces within template string variables.  It makes it challenging to use later on.  It's better to do the spacing inside a template string itself then from within a variable.

```javascript
// not preferred
${radioTrendAltitude} ${currentWaypointRadioAltitude}${isExpeditingString}

// preferred
${radioTrendAltitude} ${currentWaypointRadioAltitude} ${isExpeditingString}
```

The good example reads like a sentence would. the vars themselves don't contain spaces because that is accomplished within the string itself.  The vars behave predictably and could be rearranged without worry of hidden spacing.  That isn't so easy to do with the other example.